.github/workflows/unit-tests.yaml

```
@@ -32,6 +32,12 @@ jobs:
       matrix:
         gotags: [ 'GOTAGS=""', 'GOTAGS=release' ]
     runs-on: ubuntu-latest
+    # Skip release build tests on PRs unless ci-release-build label is present
+    # Skipped jobs satisfy required status checks
+    if: |
```

actionlint (1.7.12)

[error] context "matrix" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)